src/PoolCaller.sol

```diff
@@ -32,38 +32,6 @@ library PoolCaller {
         }
     }
 
-    /// @dev Equivalent to `IUniswapV3Pool.feeGrowthGlobal0X128`
-    /// @param pool Uniswap v3 pool
-    function feeGrowthGlobal0X128(V3PoolCallee pool) internal view returns (uint256 f) {
```

Comment:
Most of these wrappers are only different in the "selector" it uses. Does it make sense to create a base function that takes selector as an argument? And all other wrapper functions can simply call it by passing in the desired selector. This way, we don't have to repeat the assembly part multiple times.

Reply:
Shrewd if it doesn't require extra internal `JUMP`.

Reply:
Okay. But backtester doesn't really need to be that gas optimized. No strong opinion. Up to you.